src/web/calculators/shared/components/KgKvKaDisplay.tsx

```diff
@@ -0,0 +1,32 @@
+import Measurement from "common/models/Measurement";
+import SingleInputLine from "../../../../common/components/io/inputs/SingleInputLine";
+import MeasurementOutput from "../../../../common/components/io/outputs/MeasurementOutput";
+import KvKaDisplay, { KvKaDisplayProps } from "./KvKaDisplay";
+
+export interface KgKvKaDisplayProps extends KvKaDisplayProps {
+  kG: Measurement;
+}
+
+export default function KgKvKaDisplay({
+  kG,
+  kV,
+  kA,
+  angular,
+}: KgKvKaDisplayProps): JSX.Element {
+  return (
+    <>
+      <SingleInputLine
+        label="Estimated kG"
+        id="kG"
+        tooltip="Gravity feedforward constant of the mechanism (assumes vertical orientation for linear mechanisms)."
+      >
+        <MeasurementOutput
+          stateHook={[kG, () => undefined]}
```

Comment:
upon further review, a statehook of `[measurement, () => undefined]` works but only for output fields. been a while since I wrote the i/o code :)